sectioning.asciidoc

```diff
@@ -1,13 +1,19 @@
-The projects preferred markup format is http://asciidoctor.org[Asciidoc].  
-Asciidoc has a lot of great features, but denoting chapter and title information is the first step in making beautiful ebooks.
+Sectioning
+----------
 
+The project's preferred markup format is http://asciidoctor.org[Asciidoc].  
+Asciidoc has a lot of great features, but denoting title and chapter information is the first step in making beautiful ebooks.
 
-The title of the book should have a row of dashes underneath.  This turns into a Level one header +<h1>+
+== Titles
+The title of the book should have a row of hyphens underneath.  This turns it into a level-one header (+<h1>+ in HTML).
 
   Alice in Wonderland
   -------------------
 
-Chapters should have two equals signs.  This turns into an +<h2>+ tag in html
+Subtitles are not possible.
+
```

Comment:
Good point. Although, I've just noticed this: http://asciidoctor.org/docs/user-manual/#subtitle-partitioning